lms/models/application_instance.py

```diff
@@ -19,6 +19,7 @@ class ApplicationSettings(JSONSettings):
         JSONSetting("blackboard", "groups_enabled", asbool),
         JSONSetting("canvas", "sections_enabled", asbool),
         JSONSetting("canvas", "groups_enabled", asbool),
+        JSONSetting("canvas", "file_picker_folders", asbool),
```

Comment:
```suggestion
        JSONSetting("canvas", "filepicker_folders", asbool),
```

I think `filepicker_folders` is more readable (separates the two parts of the name)

Reply:
We use file_picker across the code base so I went for that, eg: `lms/resources/_js_config/file_picker_config.py`

Reply:
I guess `*_enabled` would be consistent with other boolean feature flag setting names, e.g. `file_picker_folders_enabled` or `folders_enabled`